### PR TITLE
bump debian build details 4.9.1

### PR DESCRIPTION
**Pull request by un1versal**
@AlexanderP 

Do we need anything else to build current nightly for debian/ubuntu?

### CODE REVIEW

**Comment by AlexanderP** [comment]
Hi. 
It is necessary to fix:

build_debian.sh:
 test -z ${dat} && dat="git-6087-$(git describe --always)" \

**Comment by un1versal** [comment]
you mean test -z ${dat} && dat="git-6086-$(git describe --always)" \

is that right?

@AlexanderP so will your build system pick up these changes? will we need to bump this every time we want debain/ubuntu builds from your ppa?

**Comment by AlexanderP** [comment]
Yes, The ppa I need to make changes to scripts

**Comment by un1versal** [comment]
> 6087

@AlexanderP I dont understand the build is +kvirc (4:4.9.1~git-6086-g30d4263-1) unstable; urgency=medium

Do I also need to put 6087 in changelog?

My kvirc build (compiled says)

```
KVIrc 4.9.1 'Aria'

Runtime Info:
System name: Linux 3.13.0-71-generic
System version: #114-Ubuntu SMP Tue Dec 1 02:34:22 UTC 2015
Architecture: x86_64
Qt version: 5.2.1
Qt theme: fusion

Build Info:
Build date: 2016-01-08 16:26:12 UTC
Sources date: 20160102
Revision number: git-6086-g30d4263
System name: Linux-3.13.0-71-generic
CPU name: x86_64
Build command: /usr/bin/cmake
Build flags: 
   MANDIR=man
   CMAKE_INSTALL_PREFIX=/usr
   LIB_SUFFIX=/x86_64-linux-gnu
   Threads=POSIX
Compiler name: /usr/bin/c++
Compiler flags: N/A
Qt version: 5.2.1
```

File: dist/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,12 @@
+kvirc (4:4.9.1~git-6086-g30d4263-1) unstable; urgency=medium
+
+  * Compile
+  * Branch: master
+  * Commit: 30d426306bd04407480b2521167e4d008837f2b0
+  * Date: 1452270458
+
+ -- Alexander Pozdnyakov <almipo@mail.ru>  Fri, 8 Jan 2016 16:29:47 GMT
```
Comment (AlexanderP) [inline dist/debian/changelog]: `-- Alexander Pozdnyakov <almipo@mail.ru>  Fri, 8 Jan 2016 16:29:47 +0300`

**Comment by AlexanderP** [comment]
It only travis.

**Comment by un1versal** [comment]
is this OK now?

**Comment by AlexanderP** [comment]
changelog - 6086
script - 6087

**Comment by un1versal** [comment]
> changelog - 6086
> script - 6087

OK why is that? I dont understand it, and trying to understand...

**Comment by AlexanderP** [comment]
Assembled version must be higher.

**Comment by AlexanderP** [comment]
Build from WANT_KDE=YES - error

`[ 51%] Building CXX object src/kvirc/CMakeFiles/kvirc.dir/ui/KviInputHistory.cpp.o
/home/travis/build/kvirc/KVIrc/src/kvirc/ui/KviInput.cpp: In member function ‘virtual void KviInput::keyPressEvent(QKeyEvent*)’:
/home/travis/build/kvirc/KVIrc/src/kvirc/ui/KviInput.cpp:253:27: warning: unused variable ‘pYesButton’ [-Wunused-variable]
          QAbstractButton *pYesButton = pMsgBox.addButton(__tr2qs("Yes"), QMessageBox::YesRole);
                           ^
[ 51%] Building CXX object src/kvirc/CMakeFiles/kvirc.dir/ui/KviIpEditor.cpp.o
/home/travis/build/kvirc/KVIrc/src/kvirc/ui/KviInputEditor.cpp: In member function ‘void KviInputEditor::rebuildTextBlocks()’:
/home/travis/build/kvirc/KVIrc/src/kvirc/ui/KviInputEditor.cpp:637:11: warning: unused variable ‘iNextChar’ [-Wunused-variable]
       int iNextChar = KviControlCodes::getUnicodeColorBytes(spb->szText,p - spb->szText.unicode(),&uFore,&uBack);
           ^
/home/travis/build/kvirc/KVIrc/src/kvirc/ui/KviInputEditor.cpp: In member function ‘virtual void KviInputEditor::drawContents(QPainter*)’:
/home/travis/build/kvirc/KVIrc/src/kvirc/ui/KviInputEditor.cpp:844:6: warning: unused variable ‘iCurChar’ [-Wunused-variable]
  int iCurChar = 0;
      ^
/home/travis/build/kvirc/KVIrc/src/kvirc/ui/KviInputEditor.cpp: In member function ‘virtual QVariant KviInputEditor::inputMethodQuery(Qt::InputMethodQuery) const’:
/home/travis/build/kvirc/KVIrc/src/kvirc/ui/KviInputEditor.cpp:1778:8: error: ‘ImEnabled’ is not a member of ‘Qt’
   case Qt::ImEnabled:
        ^
/home/travis/build/kvirc/KVIrc/src/kvirc/ui/KviInputEditor.cpp:1781:8: error: ‘ImHints’ is not a member of ‘Qt’
   case Qt::ImHints:
        ^
[ 51%] Building CXX object src/kvirc/CMakeFiles/kvirc.dir/ui/KviIrcToolBar.cpp.o
make[2]: *** [src/kvirc/CMakeFiles/kvirc.dir/ui/KviInputEditor.cpp.o] Error 1`

**Comment by un1versal** [comment]
> Assembled version must be higher.

OK gotcha, thanks for explaining.

> Build from WANT_KDE=YES - error

Idk about that, @wodim can we fix that?

**Comment by ctrlaltca** [comment]
https://github.com/kvirc/KVIrc/commit/cad079d188f25c90b1c9a6e3add73c2c782febbb#diff-c3aa4a4955c05f5ddf447e38c25b5fd6R1778 is qt5-only, needs a proper `#ifdef`

Also, the travis script is broken; the build errored out, but the script is reporting success: https://travis-ci.org/kvirc/KVIrc/jobs/101120874

**Comment by un1versal** [comment]
> cad079d#diff-c3aa4a4955c05f5ddf447e38c25b5fd6R1778 is qt5-only, needs a proper #ifdef

Thats above my paygrade, would you mind? :grinning: 

> Also, the travis script is broken; the build errored out, but the script is reporting success: https://travis-ci.org/kvirc/KVIrc/jobs/101120874

No idea, maybe @Dessa or @DarthGandalf know what to do?

**Comment by un1versal** [comment]
@AlexanderP can this be merged now?... the other issues are unrelated and can be fixed separately...

**Comment by AlexanderP** [comment]
Yes.

**Comment by un1versal** [comment]
thanks

**Pull request merged**